faketree/faketree.go

```diff
@@ -476,6 +504,9 @@ func (opts *Flags) Parse(argv []string) ([]string, error) {
 			"specify non standard options. Do so at your own risk, as faketree may no longer work.")
 	fs.BoolVar(&opts.Wait, "wait", opts.Wait, "Wait for all children of this process to die before returning.")
 	fs.BoolVar(&opts.Propagate, "propagate", opts.Propagate, "Take control of signal propagation - see help screen for more details.")
+	fs.DurationVar(&opts.Timeout, "timeout", opts.Timeout,
```

Comment:
Love the idea - recommend naming the flag more specifically than `timeout`, to address the confusion that could arise alluded to in your PR description. Maybe `termination-wait-timeout`?